ws-nextjs-app/pages/[service]/new_live/[id]/[[...variant]].page.tsx

```diff
@@ -0,0 +1,104 @@
+import { GetServerSideProps } from 'next';
+import { ParsedUrlQuery } from 'querystring';
+import omit from 'ramda/src/omit';
+import applyBasicPageHandlers from '#pages/utils/applyBasicPageHandlers';
+import bffFetch from '#app/routes/topic/getInitialData';
+import getAgent from '#server/utilities/getAgent';
+import getToggles from '#app/lib/utilities/getToggles/withCache';
+import { LIVE_PAGE } from '#app/routes/utils/pageTypes';
+import nodeLogger from '#lib/logger.node';
+import {
+  ROUTING_INFORMATION,
+  SERVER_SIDE_RENDER_REQUEST_RECEIVED,
+} from '#app/lib/logger.const';
+import { Services, Variants } from '#models/types/global';
+
+import LivePageLayout from './LivePageLayout';
+
+export default applyBasicPageHandlers({
+  addVariantHandling: true,
+})(LivePageLayout);
+
+interface PageDataParams extends ParsedUrlQuery {
+  id: string;
+  page?: string;
+  service: Services;
+  variant?: Variants;
+  // eslint-disable-next-line camelcase
+  renderer_env?: string;
+}
+
+const getPageData = async ({
+  id,
+  page,
+  service,
+  variant,
+  rendererEnv,
+}: PageDataParams) => {
+  const data = await bffFetch({
+    getAgent,
+    page,
+    path: `${id}${rendererEnv ? `?renderer_env=${rendererEnv}` : ''}}`,
+    service,
+    variant,
+  });
+
+  const toggles = await getToggles(service);
+
+  return { data, toggles };
+};
+
+export const getServerSideProps: GetServerSideProps = async context => {
+  const logger = nodeLogger(__filename);
+
+  const {
+    id,
+    service,
+    variant,
+    // renderer_env: rendererEnv,
```

Comment:
Do we need to uncomment this?

Reply:
Yea I'll uncomment this. The main reason I locked the environment to use `live` for the data fetch requests is so that you wouldn't need to add `renderer_env=live` if you used the pagination, as they removes any query params. Not a big deal at all though, just more of a testing usability thing 😄

Reply:
Let's keep it until we don't need it for testing purposes - there are a couple of other test specific things which we need to clean up as well as a couple of TODOs, so let's just do it then.